e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js

```diff
@@ -154,9 +154,9 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
 
   it('renders only deals that have matching fields - MIA, AIN, Draft status, Ready for check status', () => {
     const EXPECTED_DEALS = ALL_DEALS.filter(({ submissionType, status }) =>
-      status === CONSTANTS.DEALS.DEAL_STATUS.DRAFT
-      || submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA
-      || submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN);
+      (status === CONSTANTS.DEALS.DEAL_STATUS.DRAFT || status === CONSTANTS.DEALS.DEAL_STATUS.READY_FOR_APPROVAL)
+      && (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA
+      || submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN));
```

Comment:
Bring the condition to same line please.
Ideally 3 or more then each on new line.

Reply:
done